worker/uniter/runner/flush_test.go

```diff
@@ -116,194 +90,6 @@ func (s *FlushContextSuite) TestRunHookRelationFlushingSuccess(c *gc.C) {
 	})
 }
 
-func (s *FlushContextSuite) TestRunHookMetricSendingSuccess(c *gc.C) {
```

Comment:
We need to keep the test coverage & scenarios of `FlushContextSuite`. They need to be relocated & refactored to test the sendMetrics operation.